### DISCUSSION TITLE
ChatGLM3-6B在回答问题中，答案带有<|system|>

### DISCUSSION THREAD

**Discussion by xubzhlin** [zai-org/ChatGLM3#1047] [Q&A] [answered]
通过提示词，使用ChatGLM3-6B回答问题。
提示词模板如下：
`<|system|>你是一个智能客服助手，根据知识库内容简洁和专业的回答用户问题。如果无法从中得到答案，请说 “根据已知信息无法回答该问题“,不允许在答案中添加编造成分，答案请使用中文。
知识库内容"""
{{content}}
"""
<user>{{question}}
<|assistant|>
`

ChatGLM3-6b能够正确回答问题，只是回答的内容偶尔会带有`<|system|>`字样。

**Comment by zRzRzRzRzRzRzR** [comment, marked as answer]
一定要使用chat_apply_template
